cli/src/cli_util.rs

```diff
@@ -1492,12 +1492,8 @@ impl WorkspaceCommandHelper {
             &self.op_summary_template_text,
             OperationTemplateLanguage::wrap_operation,
         )
-        // TODO: "op_log" is not a great label name for op_summary. We could
-        // replace it with `operation summary` and use `operation` in
-        // colors.toml. TODO: It would be better to put labels like these
-        // consistently in the same place, e.g. `templates.toml` instead of the
-        // rust code.
-        .labeled("op_log")
+        .labeled("summary")
+        .labeled("operation")
```

Comment:
nit: shouldn't it be just `"operation"` or `"op_summary" "operation"`?

Reply:
To the degree that this was a question: I thought that "operation op_summary" is a bit redundant, and "operation summary" is better. Same for "operation op_log". Perhaps one day we'll need to formatting for all summaries, or all logs (log, op_log, evolog).

But the usefulness of that does seem a bit remote. I'm happy to do adjust it to your preference, especially since you have the rest of the formatter code fresher in your mind. The confusion about the organizing principle is part of the reason I wish all these labels were in one place.

Reply:
Well, "operation summary" sounds natural as English, but labels represent a document structure. I believe "operation id" makes more sense than ("operation ")"summary id".